app/controllers/houses_controller.rb

```diff
@@ -61,7 +61,7 @@ def current_parties
 
     @house, @parties = data.filter('http://id.ukpds.org/schema/House', 'http://id.ukpds.org/schema/Party')
     @house = @house.first
-    @parties = @parties.sort_by(:name)
+    @parties = @parties.sort_by(:member_count).reverse
```

Comment:
Will do - it seems that if we are using .reverse then we definitely need to commit some time to find an efficient way of doing this